blocks/blog-index/blog-index.css

```diff
@@ -0,0 +1,55 @@
+.blog-index .blog-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(340px, 1fr));
+    grid-gap: 10px;
+    align-items: stretch;
+
+}
+
+.blog-index-wrapper{
+    max-width:1050px !important;
+}
+
+.blog-teaser-image img{
+width:100%;
```

Comment:
Please add a `object-fit: cover;` so we don't warp the image if it doesn't fit exactly.

Please also add this for the `img` within `.blog-feature-container .blog-img`.